### PR TITLE
Modification on CBL  and modification on FMP

### PR DESCRIPTION
**Pull request by jgomezr**
Modification on CBL model to conect this with CBL2 and modification on FMP objects to improve object model, and first trigger to put information from cbl old model to CBL2 new model

### CODE REVIEW

File: src/triggers/CBLlot.trigger [hunk 1]
```diff
@@ -0,0 +1,18 @@
+trigger CBLlot on CBL_Lot__C(after insert){
+  for (CBL_Lot__C newLot : Trigger.new) {
+    if (newLot.CBL2_Lot__c <> null) {
+      for(CBL2_Lot__c cbl2lot : [SELECT lotArea__c, lotName__c, lotAreaUnits__c, lotCode__c, Lot_Code_Guat__c, lotCoffeeVariety__c, Lot_Latitude__c, Lot_Longitude__c, lotNumberOfPlants__c, lotPlantingDate__c, Submission__c FROM CBL2_Lot__c WHERE Id = :newLot.CBL2_Lot__c]){
```
Comment (gokubi) [inline src/triggers/CBLlot.trigger]: This isn't bulk-safe. Will cause a problem when a batch of CBL_Lot__c objects are inserted. Should do one soql for the entire batch.

Comment (jgomezr) [inline src/triggers/CBLlot.trigger]: the records to this object will not enter to batch, the source of this object records are surveys taken by ckw in the field

Comment (lc0rp) [inline src/triggers/CBLlot.trigger]: By batch safe, Steve means that if there are more than 100 items in Trigger.new, the trigger will fail.

How about using an "IN" statement and two nested "for" loops? That'll make it more batch-safe.

``` apex
lot_ids = new List<Id>();
for (CBL_Lot__C newLot : Trigger.new) {
       lot_ids.add(newLot.id);
}

for(CBL2_Lot__c[] cbl2lots : [SELECT lotArea__c, lotName__c, lotAreaUnits__c, lotCode__c, Lot_Code_Guat__c, lotCoffeeVariety__c, Lot_Latitude__c, Lot_Longitude__c, lotNumberOfPlants__c, lotPlantingDate__c, Submission__c FROM CBL2_Lot__c WHERE Id IN :lot_ids]){
       for (CBL2_Lot__c cbl2lot: cbl2lots) {
              // Do what you want with the lots;
              ...
       }
} 
```

**Comment by lc0rp** [comment]
There are some conflicts that need to be resolved...

**Comment by jgomezr** [comment]
Changes make on trigger to be more bulk safe, check the documentation for github to check for conflicts, and review the code of objects, changes in the objects in this pull request are those that must be

**Comment by jgomezr** [comment]
New changes on FMP no need this pull request.

**Pull request closed without merging**